lib/usb/usb_standard.c

```diff
@@ -247,19 +248,30 @@ static int usb_standard_set_configuration(usbd_device *usbd_dev,
 					  struct usb_setup_data *req,
 					  uint8_t **buf, uint16_t *len)
 {
-	int i;
+	uint8_t i;
```

Comment:
though not a big deal, but i usually use unsigned values for indexing array.
also, bNumInterfaces is uint8_t and imo using int will only add confusion